public/Day 70/index.html

```
@@ -1,118 +1,124 @@
+                                <th>Value</th>
+                                <th>Change</th>
+                            </tr>
+                        </thead>
+                        <tbody></tbody>
+                    </table>
+                </div>
+
+                <div class="custom-data-section">
+                    <div class="input-group">
+                        <label>Labels</label>
+                        <input id="custom-labels" value="Jan, Feb, Mar">
```

HTMLHint (1.9.1)

[warning] The empty tag : [ input ] must be self closed.

(tag-self-close)